solana/program_v2/programs/gateway_v2/src/types/fees.rs

```diff
@@ -0,0 +1,43 @@
+use crate::util::{OnChainSize, OC_SIZE_PUBKEY, OC_SIZE_U16};
```

Comment:
@rado0x54 This folder was created copying from what Brett did... I believe this should be part of "state/netowrk.rs" instead?